### PR TITLE
Created guide to migrate users to Umbraco Cloud

### PR DESCRIPTION
**Pull request by jonat123**
Created a new guide about how to migrate users to Umbraco Cloud when you are migrating an existing site to Umbraco Cloud, based on feedback from one of our gold partners.

### CODE REVIEW

File: Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/index.md [hunk 1]
```diff
@@ -146,11 +147,11 @@ That's it! Now that you've merged your existing site with the local clone of the
 * All Document Types, Templates, Stylesheets, etc, are in the backoffice
 
 :::note
-Umbraco Identity (Umbraco ID) is the single sign-on (SSO) feature across all Umbraco Cloud services and is required to access any project pages as well as backoffices.
+Umbraco Identity (Umbraco ID) is the single sign-on (SSO) feature across all Umbraco Cloud services.
```
Comment (jonat123) [inline Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/index.md:150]:
```suggestion
Umbraco Identity (Umbraco ID) is the Single Sign-On (SSO) feature across all Umbraco Cloud services.
```

File: Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/Migrating-Users-To-Umbraco-Cloud/index.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+4. Click ***next*** then click ***advanced***
+5. Find ***“Type of data to script”*** and choose ***“Data only”***
+6. Choose where to save the script and then save it.
+7. Connect to your [Umbraco Cloud Database](/Umbraco-Cloud/Databases/Cloud-Database/index.md)
+8. Run the script on your Umbraco Cloud Database
+9. Restart your Umbraco Cloud environment
+
+### Finishing the Migration
+
+Once the script has been run on your Umbraco Cloud Database it is now time for the users to login to the Cloud backoffice.
```
Comment (eshanrnh) [inline Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/Migrating-Users-To-Umbraco-Cloud/index.md:54]:
```suggestion
Once the script has been run on your Umbraco Cloud Database, it is time for the users to log in to the Cloud backoffice.
```

File: Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/Migrating-Users-To-Umbraco-Cloud/index.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+The user will be promted to login, they should use the password from the on-premise installation.
```
Comment (eshanrnh) [inline Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/Migrating-Users-To-Umbraco-Cloud/index.md:56]:
```suggestion
The user will be prompted to log in and they should use the password from the on-premises installation.
```

File: Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/Migrating-Users-To-Umbraco-Cloud/index.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+Once the user logs in they will be asked to enter their email to receive a token and verify their user in Umbraco ID.
+
+Enter the code to verify the user with Umbraco ID.
+
+Once the user has been verified they will be redirected to the backoffice and their user has been migrated to Umbraco ID.
```
Comment (eshanrnh) [inline Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/Migrating-Users-To-Umbraco-Cloud/index.md:62]:
```suggestion
Once the user has been verified, they will be redirected to the backoffice and their user has been migrated to Umbraco ID.
```

File: Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/index.md [hunk 5]
```diff
@@ -146,11 +147,11 @@ That's it! Now that you've merged your existing site with the local clone of the
 * All Document Types, Templates, Stylesheets, etc, are in the backoffice
 
 :::note
-Umbraco Identity (Umbraco ID) is the single sign-on (SSO) feature across all Umbraco Cloud services and is required to access any project pages as well as backoffices.
+Umbraco Identity (Umbraco ID) is the Single Sign-On (SSO) feature across all Umbraco Cloud services.
 
-Any users that you might have had on your existing Umbraco site will be migrated over to the local clone of the Cloud project along with the database. These users, however, will not be able to access the Cloud environments of the project or any of the backoffices associated with those environments.
+It is required to access any project pages as well as backoffices.
```
Comment (eshanrnh) [inline Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/index.md:152]:
```suggestion
It is required to access any project pages as well as the backoffice.
```

File: Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/index.md [hunk 6]
```diff
@@ -146,11 +147,11 @@ That's it! Now that you've merged your existing site with the local clone of the
...
 
-For the users to continue having access to the project, they will need to be re-invited either as a [Team Member](../../Set-Up/Team-Members) on a project level or as a [User](../../Set-Up/Users-on-Cloud) to the backoffice of one or more Cloud environments.
+To migrate the users from your on-premise site to Umbraco Cloud and Umbraco ID follow the [Migrate Users to Cloud guide](/Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/Migrating-Users-To-Umbraco-Cloud/index.md).
```
Comment (eshanrnh) [inline Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/index.md:154]:
```suggestion
To migrate the users from your on-premises site to Umbraco Cloud and Umbraco ID, follow the [Migrate Users to Cloud guide](/Umbraco-Cloud/Getting-Started/Migrate-Existing-Site/Migrating-Users-To-Umbraco-Cloud/index.md) article.
```

**Review by eshanrnh — CHANGES_REQUESTED** [review]
Looks good, @jonat123 🙌 Just a couple of suggestions.

**Review by eshanrnh — APPROVED** [review]
Looks good, @jonat123 🙌 Merging it in.

**Pull request merged**